src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import { useParams } from 'react-router-dom';
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  person: Person;
+  mother?: Person;
+  father?: Person;
+}
+
+export const PersonInfo: React.FC<Props> = ({ person }) => {
+  const {
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  } = person;
+
+  const { personSlug } = useParams();
+  const isPersonSelected = personSlug === slug;
+
+  return (
+    <tr
+      data-cy="person"
```

Comment:
Remove the 'key' prop from the 'tr' element as it is not necessary in this context.